### ISSUE TITLE
Feature Request: The visiblity of ALT text overlay

### ISSUE THREAD

**Issue by Russssty** [pachli/pachli-android#2171]
**Is your feature request related to a problem? Please describe.**
When playing a video attached to a post, the ALT text overlays the media. When descriptions approach the 1,500-character limit, the first several seconds of the video can be completely obscured by the text overlay, making it impossible to see the content.

**Describe the solution you'd like**
The visibility of the ALT text overlay should be configurable in the settings. I suggest keeping the default value as "On" (to maintain current accessibility standards) while allowing users to toggle it off if they prefer. In this case, the ALT text would still be accessible via the "ALT" badge.
